### PR TITLE
dropdown,input elements have been updated to grasp screenreaderconten…

### PR DESCRIPTION
**Pull request by MokireddySampath**
…t for all the elements

[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/1440?feature.someFeatureFlagYouMightNeed=true)

defect- https://msdata.visualstudio.com/CosmosDB/_workitems/edit/2276933/

### CODE REVIEW

File: src/Explorer/Panes/Tables/AddTableEntityPanel.tsx [hunk 1]
```diff
@@ -254,11 +254,14 @@ export const AddTableEntityPanel: FunctionComponent<AddTableEntityPanelProps> =
...
           return (
             <TableEntity
+              indexofelement={index}
               key={"" + entity.id + index}
               isDeleteOptionVisible={entity.isDeleteOptionVisible}
-              entityTypeLabel={index === 0 && dataTypeLabel}
+              entityTypeLabel={dataTypeLabel}
               entityPropertyLabel={index === 0 && attributeNameLabel}
               entityValueLabel={index === 0 && attributeValueLabel}
+              entityValueAriaLabel={attributeValueLabel}
```
Comment (victor-meng) [inline src/Explorer/Panes/Tables/AddTableEntityPanel.tsx:263]: The `attributeValueLabel` and `attributeNameLabel` are constants from `EntityTableHelper.tsx`. Why do you need to pass them via parameters? You can just import them from `EntityTableHelper.tsx`?

File: src/Common/TableEntity.tsx [hunk 2]
```diff
@@ -100,6 +105,22 @@ export const TableEntity: FunctionComponent<TableEntityProps> = ({
     }
     return "string";
   };
+  const dropdownandlabelStyles: Partial<IDropdownStyles> = {
```
Comment (victor-meng) [inline src/Common/TableEntity.tsx:108]: What's the purpose of this style?

**Review by victor-meng — CHANGES_REQUESTED** [review]
A second scrollbar appears in the panel now and there's extra spaces at the end.

image

File: src/Common/TableEntity.tsx [hunk 3]
```diff
@@ -43,13 +44,17 @@ export interface TableEntityProps {
...
   entityTypeLabel,
   entityPropertyLabel,
   isDeleteOptionVisible,
   entityProperty,
   selectedKey,
   entityPropertyPlaceHolder,
   entityValueLabel,
+  entityValueAriaLabel,
+  entityPropertyAriaLabel,
+
```
Comment (victor-meng) [inline src/Common/TableEntity.tsx:57]: Remove extra space

**Review by vchske — APPROVED** [review]

**Review by jawelton74 — APPROVED** [review]

**Pull request merged**